izanami-frontend/src/components/MultiSelect.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import Select, { components } from "react-select";
+import { customStyles } from "../styles/reactSelect";
+
+export type Option = {
+  value: string;
+  label: string;
+  state: boolean;
+};
+const MultiSelect = (props: any) => {
```

Comment:
Since this component await many props whose format may not be obvious, could you define exact type for this props ?